plat/hisilicon/hikey960/hikey960_pm.c

```diff
@@ -295,6 +296,14 @@ static const plat_psci_ops_t hikey960_psci_ops = {
 int plat_setup_psci_ops(uintptr_t sec_entrypoint,
 			const plat_psci_ops_t **psci_ops)
 {
+	unsigned int id;
+
+	hikey960_read_boardid(&id);
```

Comment:
Static analysis error - MISRA C-2012 Rule 17.7 violation: Return value of non-void function "hikey960_read_boardid()" is not used.

This method does return an error code - could return error from hikey960_read_boardid()?